### ISSUE TITLE
ENH: Time estimate for file conversion

### ISSUE THREAD

**Issue by Jython1415** [Jython1415/zolltools#12]
# Problem
This is because it is hard to figure out how much longer I have to let the program run before the conversion will process.

# Solution
- [ ] Convert a small part of the table and see how long that takes
- [ ] Verify that small part of the table and see how long that takes
- [ ] Figure out how big the table is and output the the estimate based on that
- Optional: Track conversion speed throughout the process to see if it changes for different parts of the file

**Comment by Jython1415** [comment]
> * Optional: Track conversion speed throughout the process to see if it changes for different parts of the file

It may be better to make this it's own issue once the initial time estimate is complete. I could also add a comparison at the end to see how good the initial estimate was compared to the actual result.

**Comment by Jython1415** [comment]
duplicate of #2
